common/logger/src/lib.rs

```diff
@@ -141,6 +141,7 @@ pub mod prelude {
 }
 pub mod json_log;
 
+mod kv;
```

Comment:
this whole entry point (crate / module import / re-export) is a mess, care to try to clean it up in one of your future commits / PRs?

Reply:
Yep that's on my list of things to do, I also want to split this into a crate to be used in libraries (for instrumentation) and a crate which actually implements the logger for binaries. This should allow us to make the dep footprint a little more palatable too